IOSAccessAssessment/Services/ChangesetService.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ChangesetService.swift
+//  IOSAccessAssessment
+//
+//  Created by Mariana Piz on 11.11.2024.
+//
+
+import Foundation
+
+struct NodeData {
+    let latitude: Double
+    let longitude: Double
+}
+
+class ChangesetService {
+    
+    private enum Constants {
+        static let baseUrl = "https://osm.workspaces-stage.sidewalks.washington.edu/api/0.6"
+        static let workspaceId = "168"
+    }
+    
+    static let shared = ChangesetService()
+    private init() {}
+    
+    private let accessToken = KeychainService().getValue(for: .accessToken)
+    private(set) var changesetId: String?
+    
+    func openChangeset(completion: @escaping (Result<String, Error>) -> Void) {
+        guard let url = URL(string: "\(Constants.baseUrl)/changeset/create"),
+              let accessToken
+        else { return }
+        
+        var request = URLRequest(url: url)
+        request.httpMethod = "PUT"
+        request.setValue("Bearer \(accessToken)", forHTTPHeaderField: "Authorization")
+        request.setValue(Constants.workspaceId, forHTTPHeaderField: "X-Workspace")
+        request.setValue("application/xml", forHTTPHeaderField: "Content-Type")
+        
+        guard let xmlData = """
+            <osm>
+                <changeset>
+                    <tag k="created_by" v="GIG 1.0(4)" />
```

Comment:
In the new PR that we raise for change sets (I think we will have to considering the conflicts), we should change this to match our own arguments. 
Maybe for now: created_by = iOSPointMapper
comment can be left blank for now.

Reply:
Both "created_by" and "comment" are from documentation (4 b Open a changeset): https://docs.google.com/document/d/1qPSW5pYmVb-RXOiryBm452sLfTvFl38f_TCO4SElkSY/edit?tab=t.0

Reply:
I resolved merge conflicts

Reply:
Yes, but that example input is for GoInfoGame. We are a different creator, so we should be having our own identifier.

Reply:
Ah cool. Let's proceed with this PR then